cmake/policies.cmake

```diff
@@ -0,0 +1,3 @@
+# cmake --help-policy CMP0054
```

Comment:
Better describe what does the policy do

Reply:
you can always write this command and read about it

Reply:
When you're reading source files, you're probably don't want to switch to the terminal and read big mans